code/src/item_effect.h

```diff
@@ -18,6 +18,7 @@ void ItemEffect_GiveMagic(SaveContext* saveCtx, s16 arg1, s16 arg2);
 void ItemEffect_GiveDoubleMagic(SaveContext* saveCtx, s16 arg1, s16 arg2);
 void ItemEffect_GiveFairyOcarina(SaveContext* saveCtx, s16 arg1, s16 arg2);
 void ItemEffect_GiveSong(SaveContext* saveCtx, s16 questBit, s16 arg2);
+void ItemEffect_GiveUpgrade(SaveContext* saveCtx, s16 questBit, s16 arg2);

```

Comment:
doesnt look the the name `questBit` is right. I would say either change it to just `arg1`, or rename `questBit` to `value` and `arg2` to `upgrade`.